modules/apps/translation/translation-api/src/main/java/com/liferay/translation/translator/TranslatorRegistry.java

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.translation.translator;
+
+import com.liferay.portal.kernel.module.configuration.ConfigurationException;
+
+import java.util.List;
+
+/**
+ * @author Alicia Garcia
+ */
+public interface TranslatorRegistry {
+
+	public Translator getFirstTranslator(long companyId)
```

Comment:
Can we simply call this `getTranslator` or `getCompanyTranslator` (in case later we add support for group scoped translators)? Pick your poison ;-)

Reply:
![](https://media1.giphy.com/media/NW0SGKBZs28eI/giphy.gif)